pipelines/edx/mfe-frontend-app-learning-qa.yml

```diff
@@ -59,13 +59,24 @@ jobs:
         SUPPORT_URL: https://mitx-micromasters.zendesk.com/hc/en-us/requests/new
         USER_INFO_COOKIE_NAME: edx-user-info
         SESSION_COOKIE_DOMAIN: .mitxonline.mit.edu
+        ABOUT_US_URL: https://mitxonline.mit.edu/about-us/
+        PRIVACY_POLICY_URL: https://mitxonline.mit.edu/privacy-policy/
+        HONOR_CODE_URL: https://mitxonline.mit.edu/honor-code/
+        TERMS_OF_SERVICE_URL: https://mitxonline.mit.edu/terms-of-service/
+        Contact: mailto:mitxonline-support@mit.edu
```

Comment:
is `Contact` the right key?

Reply:
Yes, I missed the missed the CAPS on the key somehow. I will update it if we need to change the code after QA test.